### PR TITLE
Add move PRs to .git-blame-ignore-revs

### PR DESCRIPTION
**Pull request by maxentile**
To be completed and merged after https://github.com/proteneer/timemachine/pull/614 and https://github.com/proteneer/timemachine/pull/615

### CODE REVIEW

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -1,2 +1,27 @@
...
+
+# Move scripts/ to attic/scripts
+1a721dd3f05d6011cf028b0588e066682d38ba59
+
+# Move testsystems/ to timemachine/testsystems/
+#   and datasets/ to timeamchine/datasets/
+1e414a0e778948441cd3d01e51c62380230c96ea
+
+# Move parallel/ to timemachine/parallel
+4b814f2ee02be99a4ee4a4e8a4b55fc03bc28adf
```
Comment (maxentile) [inline .git-blame-ignore-revs:27]: @badisa : confirm if you would like https://github.com/proteneer/timemachine/commit/4b814f2ee02be99a4ee4a4e8a4b55fc03bc28adf added to `.git-blame-ignore-revs`?

Comment (badisa) [reply .git-blame-ignore-revs:27]: That would be great!

**Review by badisa — APPROVED** [review]

**Review by mcwitt — APPROVED** [review]

**Comment by mcwitt** [comment]
Would it make sense to add https://github.com/proteneer/timemachine/pull/613/commits/2200b6c482d41b49819a4081501e016328dbb839 to the list here?

**Comment by mcwitt** [comment]
> Would it make sense to add https://github.com/proteneer/timemachine/commit/2200b6c482d41b49819a4081501e016328dbb839 to the list here?

Ack, sorry, the commit hash changed before this was merged. The current one (in `master`) is https://github.com/proteneer/timemachine/commit/61e068297ce26083b1168535ba908f9987696605

**Review by proteneer — APPROVED** [review]
lgtm!

**Pull request merged**